internal/cmd/local/prerun.go

```diff
@@ -0,0 +1,62 @@
+package local
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+//// persistentPreRunLocal is extracted from the cobra.Command initialization for reading purposes only
```

Comment:
If this has now been migrated over to another location let's clean clean this up

Reply:
I had already done this, forgot to push it last night.